lib/routes/shared/initial_walkthrough.dart

```diff
@@ -217,7 +242,10 @@ class InitialWalkthroughPageState extends State<InitialWalkthroughPage>
                               return BetaWarningDialog();
                             }).then((approved) {
                           if (approved) {
-                            _proceedToRegister();
+                            widget._registrationBloc.userActionsSink.add(_updateSecurityModelAction);
```

Comment:
Better not to save `_updateSecurityModelAction ` as class variable. Just create an action here for resetting the key.

Reply:
Removed the usage of _updateSecurityModelAction variable [here](https://github.com/breez/breezmobile/pull/173/commits/4c12fa91203d1b146b9a1ad1678c2ebafec1a1e2)